src/Sarif/Visitors/InsertOptionalDataVisitor.cs

```diff
@@ -150,14 +150,17 @@ public override Artifact VisitArtifact(Artifact node)
                         }
 
                         int length = node.Length;
-                        node = Artifact.Create(uri, _dataToInsert, encoding: encoding);
+
+                        if(_run.OriginalUriBaseIds != null) node = Artifact.Create(uri, _dataToInsert, encoding: encoding);
```

Comment:
need a space after if. I think i would have assigned a URI to a variable here using the ternary conditional operator and then called Artifact.Create. Alternately, our coding convention states that if/else bodies must have bracing even in cases where they are expressed on a single line.

// Cleaner IMO
Uri uri = run.OriginalUriBaseIds != null ? uri : fileLocation.Uri;
Artifact.Create(fileLocation.Uri, _dataToInsert, encoding: encoding);

// Your pattern corrected for style. Note that there is little or no other use of this pattern in the code.
// We do use single line if statements but generally not when there's an else clause.
if (_run.OriginalUriBaseIds != null) { node = Artifact.Create(uri, _dataToInsert, encoding: encoding); }
else node = { Artifact.Create(fileLocation.Uri, _dataToInsert, encoding: encoding); }